ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/insert-values.json

```
@@ -431,6 +431,36 @@
       "outputs": [
         {"topic": "test_topic", "timestamp": 1234, "key": {"F1": "key"}, "value": {"ID": 10}}
       ]
+    },
+    {
+      "name": "verify INSERT VALUES fails for ROWPARTITION (matching types)",
```

```suggestion
      "name": "verify INSERT VALUES fails for ROWPARTITION",
```
and similarly for the test below -- it's unclear to readers what "matching types" means and I don't think it adds much.